### ISSUE TITLE
[Internal] Bintray uploading broken with new kts scripts

### ISSUE THREAD

**Issue by samtstern** [firebase/FirebaseUI-Android#1330]
I am working on this, but filing it for good measure.  Here's what I see for the logs when trying to publish the `common` module:

```
> Task :common:bintrayUpload
Caching disabled for task ':common:bintrayUpload': Caching has not been enabled for the task
Task ':common:bintrayUpload' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
Gradle Bintray Plugin version: 1.8.0
maven plugin was not applied, no pom will be uploaded.
Uploading to https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/com/firebaseui/common/4.0.1/common-4.0.1-javadoc.jar...
Uploaded to 'https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/com/firebaseui/common/4.0.1/common-4.0.1-javadoc.jar'.
Uploading to https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/com/firebaseui/common/4.0.1/common-4.0.1-sources.jar...
Uploaded to 'https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/com/firebaseui/common/4.0.1/common-4.0.1-sources.jar'.
Uploading to https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/com/firebaseui/common/4.0.1/common-4.0.1.aar...
Uploaded to 'https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/com/firebaseui/common/4.0.1/common-4.0.1.aar'.
Uploading to https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/value: com/firebaseui/value: firebase-ui-common/value: 4.0.1/value: firebase-ui-common-value: 4.0.1.aar...
Uploaded to 'https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/value: com/firebaseui/value: firebase-ui-common/value: 4.0.1/value: firebase-ui-common-value: 4.0.1.aar'.
Uploading to https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/value: com/firebaseui/value: firebase-ui-common/value: 4.0.1/value: firebase-ui-common-value: 4.0.1-javadoc.jar...
Uploaded to 'https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/value: com/firebaseui/value: firebase-ui-common/value: 4.0.1/value: firebase-ui-common-value: 4.0.1-javadoc.jar'.
Uploading to https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/value: com/firebaseui/value: firebase-ui-common/value: 4.0.1/value: firebase-ui-common-value: 4.0.1-sources.jar...
Uploaded to 'https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/value: com/firebaseui/value: firebase-ui-common/value: 4.0.1/value: firebase-ui-common-value: 4.0.1-sources.jar'.
Uploading to https://api.bintray.com/content/firebaseui/firebase-ui/firebase-ui-common/4.0.1/value: com/firebaseui/value: firebase-ui-common/value: 4.0.1/value: firebase-ui-common-value: 4.0.1.pom...
```

Then it fails.

I have added some logs to the setup and can't figure out what the `value` crap is:

```
Creating maven publication commonLibrary
        group: com.firebaseui
        artifact: firebase-ui-common
        version: 4.0.1
        aar: /usr/local/google/home/samstern/Projects/firebase/firebaseui-android/common/build/outputs/aar/common-release.aar
Bintray configuration for commonLibrary
        artifact: firebase-ui-common
        pub: commonLibrary
        config: archives
POM Transformation
        src: /usr/local/google/home/samstern/Projects/firebase/firebaseui-android/common/build/publications/commonLibrary/pom-default.xml
        dst: com/firebaseui/firebase-ui-common/4.0.1/
        name: firebase-ui-common-4.0.1.pom
```

**Comment by samtstern** [comment]
Pretty sure there's some issue with the artifacts we are declaring which is causing issues here:
https://github.com/bintray/gradle-bintray-plugin/blob/master/src/main/groovy/com/jfrog/bintray/gradle/BintrayUploadTask.groovy#L368-L424

**Comment by SUPERCILEX** [comment]
So the artifactory bit from yesterday works, right? It's "just" bintray publishing that's broken? Maybe when we apply the plugin is important... I moved that around. If worst comes to worst, we can always add back a Gradle file with the old bintray stuff and then use `apply(from = "bintray-setup-tmp.gradle")`.

**Comment by samtstern** [comment]
Yeah good point.  And it does seem that artifactory works so it's just bintray,

**Comment by samtstern** [comment]
Well #1331 helped get rid of the errors, but now I get something bad still:

![image](https://user-images.githubusercontent.com/8466666/40815077-d0bd2bea-64f8-11e8-93a3-05a6c2914614.png)

You can see there is now an artifact just called `firestore` that's also being uploaded in addition to `firebase-ui-firestore`

**Comment by samtstern** [comment]
I can't look at this any more today haha I will lose my mind.  Tomorrow is a new day.

**Comment by samtstern** [comment]
Ok I got the temporary fix thing working mostly!  Enough to get `4.0.1` out the door
